packages/sdk/tasks/upgradeMarket.ts

```diff
@@ -2,7 +2,7 @@ import { TransactionReceipt } from "@ethersproject/abstract-provider";
 import { constants, Contract } from "ethers";
 import { task, types } from "hardhat/config";
 
-import { Comptroller } from "../lib/contracts/typechain";
+import {Comptroller, FuseFeeDistributor} from "../lib/contracts/typechain";
```

Comment:
Split imports here between

`../lib/contracts/typechain/Comptroller`

and

`../lib/contracts/typechain/FuseFeeDistributor`